docs/devguide/contributing.asciidoc

```diff
@@ -48,16 +48,19 @@ The Beats are Go programs, so install the latest version of
 http://golang.org/[Go] if you don't have it already. The current Go version
 used for development is Go {go-version}.
 
-The location where you clone is important. Please clone under the source
-directory of your `GOPATH`. If you don't have `GOPATH` already set, you can
-simply set it to the `go` directory in your home (`export GOPATH=$HOME/go`).
+After https://golang.org/doc/install[installing Go], set the
+https://golang.org/doc/code.html#GOPATH[GOPATH] environment variable to point to
+your workspace location, and make sure the Go `bin` directory is in your PATH.
+
+The location where you clone is important. Clone the beats repository under the
+source directory of your `GOPATH`. If you don't have `GOPATH` already set, you
```

Comment:
> Clone the beats repository under the source directory of your `GOPATH`

This sentence is a little confusing to me, as it implies to clone into `$GOPATH/src`. But when cloning one must keep the complete URL path, as is correctly shown in the following comment.

Reply:
Totally agree. The statement is ambiguous. I've tried to make it more explicit. I've also removed  the example of how to set GOPATH since we link to the docs earlier. (IMO, it doesn't make sense to tell users "set this" and then say "if it's not set, do this")